extension/sidebar-pane.js

```diff
@@ -63,6 +83,11 @@ async function _updateSubtree(selectedNode) {
  */
 async function _recursiveNodesIssuesRendering(index, nodes,
                                               skipPseudo, listEl, groupsCache) {
+  if (!_isCurrentProcess(listEl)) {
+    // New updating process is running.
+    return;
```

Comment:
Maybe here is the place where we should stop the old recursive rendering and remote the old `listEl` node.